README.md

```diff
@@ -101,21 +101,30 @@ Then just use anywhere in your code (later in the REPL session for example)  `da
 (Which indeed what that string macro expands into -- even if it has to download the data first).
 
 
+## Wildcard Use
 
+The above examples show the usage of `generate()` with a DataRepo. There are instances when the user isn’t sure of the data generators to use or if the user wants to reap maximum benefits from all the generators. Hence in order to facilitate that, you can use `generate()` without providing the DataRepo as an argument:
 
+```julia
+    generate("10.5061/dryad.74699")
```

Comment:
Don't use this as an example.
Use something with an URL.
A URL is the most common identifier.